stdlib/dis.pyi

```diff
@@ -38,25 +38,60 @@ __all__ = [
 _HaveCodeType: TypeAlias = types.MethodType | types.FunctionType | types.CodeType | type | Callable[..., Any]
 _HaveCodeOrStringType: TypeAlias = _HaveCodeType | str | bytes
 
-class Instruction(NamedTuple):
-    opname: str
-    opcode: int
-    arg: int | None
-    argval: Any
-    argrepr: str
-    offset: int
-    starts_line: int | None
-    is_jump_target: bool
+if sys.version_info >= (3, 11):
+    class Positions(NamedTuple):
+        lineno: int | None = ...
+        end_lineno: int | None = ...
+        col_offset: int | None = ...
+        end_col_offset: int | None = ...
+
+if sys.version_info >= (3, 11):
+    class Instruction(NamedTuple):
+        opname: str
+        opcode: int
+        arg: int | None
+        argval: Any
+        argrepr: str
+        offset: int
+        starts_line: int | None
+        is_jump_target: bool
+        positions: Positions | None = ...
+
+else:
+    class Instruction(NamedTuple):
+        opname: str
+        opcode: int
+        arg: int | None
+        argval: Any
+        argrepr: str
+        offset: int
+        starts_line: int | None
+        is_jump_target: bool
```

Comment:
This would be simpler with `if` statement inside the class:

```suggestion
class Instruction(NamedTuple):
    opname: str
    opcode: int
    arg: int | None
    argval: Any
    argrepr: str
    offset: int
    starts_line: int | None
    is_jump_target: bool
    if sys.version_info >= (3, 11):
        positions: Positions | None = ...
```

Reply:
Unfortunately, at least mypy doesn't like that